packages/angular/pwa/package.json

```
@@ -12,12 +12,12 @@
     "save": false
   },
   "dependencies": {
-    "@angular-devkit/schematics": "0.0.0-PLACEHOLDER",
-    "@schematics/angular": "0.0.0-PLACEHOLDER",
+    "@angular-devkit/schematics": "workspace:0.0.0-PLACEHOLDER",
+    "@schematics/angular": "workspace:0.0.0-PLACEHOLDER",
     "parse5-html-rewriting-stream": "7.0.0"
   },
   "peerDependencies": {
-    "@angular/cli": "0.0.0-ANGULAR-FW-PEER-DEP"
+    "@angular/cli": "../cli"
```

`0.0.0-ANGULAR-FW-PEER-DEP` was incorrect here, it was meant to be `^0.0.0-PLACEHOLDER`
Can't this be `workspace:^0.0.0-PLACEHOLDER`?

```suggestion
    "@angular/cli": "workspace:^0.0.0-PLACEHOLDER"
```

---

Ah sweet!